recipes/default.rb

```diff
@@ -5,38 +5,50 @@
 # Copyright 2014, 2015 Bloomberg Finance L.P.
 #
 
+node.default['nssm']['install_location'] = '%WINDIR%'
```

Comment:
NSSM is [the Non-Sucking Service Manager](https://nssm.cc/). It's also what chocolatey used to run Consul on Windows. I removed chocolatey as we can't rely on the maintainers to keep Consul up-to-date. I also found using [chef-nssm](https://github.com/dhoer/chef-nssm) was a lot cleaner.